src/sshping.cxx

```diff
@@ -127,13 +142,100 @@ void die(const char* msg) {
     exit(255);
 }
 
+#if _WIN32
+  #include <windows.h>
+  #include <conio.h>
+  #ifndef PASS_MAX
+    #define PASS_MAX 512
+  #endif
+
+double PCFreq = 0;
+
+// Replacement for the getpass UNIX method
+char *getpass(const char *prompt) {
+	char getpassbuf[PASS_MAX + 1];
+	size_t i = 0;
+	int c;
+	if (prompt) {
+		fputs(prompt, stderr);
+		fflush(stderr);
+	}
+	for (;;) {
+		c = _getch();
+		if (c != 0) {
+			if (c == '\r') {
+				getpassbuf[i] = '\0';
+				break;
+			}
+			else if (c == '\b' && i != 0) {
+				getpassbuf[i] = NULL;
+				i--;
+			}
+			else if (i < PASS_MAX && c != '\b') {
+				getpassbuf[i++] = c;
+			}
+			if (i >= PASS_MAX) {
+				getpassbuf[i] = '\0';
+				break;
+			}
+		}
+	}
+	if (prompt) {
+		fputs("\r\n", stderr);
+		fflush(stderr);
+	}
+	return _strdup(getpassbuf);
+}
+
+// Replacement for the getpid UNIX method
+DWORD getpid() {
+	return GetCurrentProcessId();
+}
+
+// Replacement for the strsep UNIX method
+char* strsep(char** stringp, const char* delim) {
+	char* start = *stringp;
+	char* p;
+	p = (start != NULL) ? strpbrk(start, delim) : NULL;
+	if (p == NULL) {
+		*stringp = NULL;
+	}
+	else {
+		*p = '\0';
+		*stringp = p + 1;
+	}
+	return start;
+}
+
+// Replacement for the clock_gettime UNIX method
+uint64_t get_time() {
+	LARGE_INTEGER li;
+	long temp = 0;
+	if (PCFreq == 0) {
+		QueryPerformanceFrequency(&li);
+		PCFreq = (double)li.QuadPart / GIGA;
+	}
+	QueryPerformanceCounter(&li);
+	return double(li.QuadPart) / PCFreq;
+}
+
+#else
+
+uint64_t GetTime() {
```

Comment:
You missed this one s/GetTime/get_time/g

Not a bad idea to compile on Windows and Linux before you push.